documents/installation-list.md

```diff
@@ -0,0 +1,47 @@
+# Installation List
+
+## For OSX Users:
+
+- Install [Brew](http://brew.sh/)
+- Optional: iTerm2
+
+## For Linux Users:  
+
+| Distro        | Base           | Package Manager | 
+|-------------|-------------|-----  
+| [Ubuntu](https://www.ubuntu.com/) | [Debian](https://www.debian.org/) | [APT](https://help.ubuntu.com/lts/serverguide/apt.html) | 
+| [Linux Mint](https://linuxmint.com/) | [Debian](https://www.debian.org/) | [APT](https://help.ubuntu.com/lts/serverguide/apt.html) |  
+
+- Ubuntu and Linux Mint are probably the two most used desktop Linux distros out there. Mint is becoming increasingly popular and in a lot of ways, is more beginner-friendly than Ubuntu. It also uses up less memory resource (the most lightweight Mint edition is XFCE), and will be better suited to slower machines.
+
+## For Windows Users:
```

Comment:
I also feel like we could mention WSL2, since a few of FAC19 ended up using it successfully with Postgres. Since dual-booting appears to be impossible for at least some hardware.

Reply:
My understanding was that those who were using WSL were having problems at Postgres week.

Were there people using straight Windows in DB week?

Reply:
At least one person got WSL2 working. The two who ended up spending ages switching to Linux were on straight Windows yeah.

Since WSL2 is still v new/experimental maybe we should wait to recommend it